MAUI/DataForm/migration.md

```diff
@@ -0,0 +1,961 @@
+---
+layout: post
+title: Migrate from Syncfusion Xamarin SfDataForm to .NET MAUI SfDataForm
+description: Learn here all about Migrating from Syncfusion Xamarin DataForm to Syncfusion .NET MAUI DataForm control and more.
+platform: MAUI
+control: SfDataForm
+documentation: ug
+---  
+
+# Migrate from Xamarin.Forms SfDataForm to .NET MAUI SfDataForm 
+
+To make the migration from the Xamarin [Xamarin SfDataForm] to [.NET MAUI SfDataForm] easier, most of the APIs from the Xamarin SfDataForm were kept in the.NET MAUI SfDataForm. However, to maintain the consistency of API naming in the.NET MAUI SfDataForm, some of the APIs have been renamed. Please find the difference in the following topics.
+
+## Namespaces 
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th></tr>
+<tr>
+<td>Syncfusion.SfDataForm.XForms</td>
+<td>Syncfusion.Maui.SfDataForm</td></tr>
+</table>
+
+## Initialize control
+
+To initialize the control, import the DataForm namespace and initialize SfDataForm as shown in the following code sample.
+
+<table>
+<tr>
+<th>Xamarin SfDataForm</th>
+<th>.NET MAUI SfDataForm</th></tr>
+<tr>
+<td>
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage
+ xmlns:dataForm="clr-namespace:Syncfusion.XForms.DataForm;assembly=Syncfusion.SfDataForm.XForms">
+        <dataForm:SfDataForm x:Name="dataForm"/>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.XForms.DataForm;
+...
+
+SfDataForm dataForm = new SfDataForm();
+this.Content = dataForm;
+
+{% endhighlight %}
+{% endtabs %}
+
+</td>
+<td>
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage
+xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm x:Name="dataForm"/>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+using Syncfusion.Maui.DataForm;
+…
+
+SfDataForm dataForm = new SfDataForm();
+this.Content = dataForm;
+
+{% endhighlight %}
+{% endtabs %}
+</td></tr>
+</table>
+
+## Classes 
+
+<table>
```

Comment:
Please all the new classes are addressed

New API - https://npmci.syncfusion.com/maui/api/internal-development/maui-dataform/api/toc.html#

Xamarin API - https://help.syncfusion.com/cr/xamarin/Syncfusion.XForms.DataForm.html

Reply:
Checked all the new classes are addressed